mautrix_facebook/example-config.yaml

```diff
@@ -90,6 +89,7 @@ bridge:
     # "nickname"
     # "own_nickname" (user-specific!)
     displayname_preference:
+    - nickname
```

Comment:
I don't really want to add things here without knowing what they do

Reply:
From my testing nickname is `None` for most people, but if you specify a nickname in a 1:1 chat it will be set (and used b/c higher in the list than `name`).

I will try to do some testing if a person has multiple nick names (in different chats).

Reply:
Right, then it definitely shouldn't be enabled by default (and could probably have a `(user-specific!)` comment). Per-room nicknames need to be implemented separately, this system currently only handles global names.